### PR TITLE
Add Badge and BadgeGroup components

### PR DESCRIPTION
**Pull request by chriszarate**
Also

- Disable smart knobs and it seems to clobber the stories imported into the docs pages. Will look into that issue.
- Pivot to `isArticle` instead of `type` in `TextGroup`.
- Start a utils repo, with `resizeImage` as the first member.

Screen Shot 2020-08-12 at 6 11 47 PM
Screen Shot 2020-08-12 at 6 11 12 PM

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ import { withA11y } from '@storybook/addon-a11y';
 import MockAppDecorator from './addons/MockAppDecorator/MockAppDecorator'
 import qzTheme from './themes/qz';
 
-addDecorator( withSmartKnobs() );
```
Comment (jshakes) [inline .storybook/preview.js:9]: I spent a while trying to figure out how to get this to work with docs, but to no avail. It does work if you declare your own knobs, but that's kind of a drag

File: package.json [hunk 2]
```diff
@@ -22,13 +22,16 @@
 		"src/**/*.jsx",
 		"src/**/*.scss"
 	],
+	"dependencies": {
+		"@quartz/js-utils": "github:Quartz/js-utils#8ab4cf9"
```
Comment (jshakes) [inline package.json:26]: 🥳

File: src/components/Badge/Badge.docs.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import { Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import Badge from './Badge';
+
+# Badge
+
+A `Badge` is a round image of predetermined size.
```
Comment (jshakes) [inline src/components/Badge/Badge.docs.mdx:6]: Could you include some examples of when you may and may not want to use this component? E.g. yes for avatars, no for editorial images

File: src/components/Badge/Badge.jsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+	// Convert size prop to image dimension. All badges are (rounded) squares.
+	const px = sizeToPx[ size ];
+
+	// Create an srcSet for 1x, 2x and 3x resolution.
+	const sizes = `${px}w`;
+	const srcSet = [ 1, 2, 3 ].map( resolution => `${resizeImage( imageUrl, px * resolution, px * resolution, true )} ${resolution}x` ).join( ', ' );
+	const src = resizeImage( imageUrl, px, px, true );
+
+	return (
+		<figure className={`${styles.container} ${styles[ size ]}`}>
```
Comment (jshakes) [inline src/components/Badge/Badge.jsx:28]: I don't know that figure is the right tag for this. This component seems more geared towards decoration that content. WDYT?

Comment (chriszarate) [reply src/components/Badge/Badge.jsx:28]: Yeah I wasn't sure either. I think just a div here is better

**Pull request merged**